### DISCUSSION TITLE
Sidebar dropdown

### DISCUSSION THREAD

**Discussion by VSevagen** [pradyunsg/furo#188] [Potential Issue] [answered]
Hi, this might be an easy solve but I can't seem to find how to allocate a dropdown to one particular section of toctree. I looked into the theme docs but doesn't seem to be there or maybe i missed it. Can someone help with that ?

**Comment by VSevagen** [comment, marked as answer]
My bad. I got the answer after some time. Turns out it had nothing to do with the theme. I just forgot to adjust the spacing after the toctree declaration.

**Reply by pradyunsg** [reply]
I do wish sometimes that reST had better error messaging. 😅
